src/actions/AuthActions.js

```diff
@@ -27,18 +43,30 @@ const attemptFacebookLogin = async ({dispatch, navigate}) => {
     if (type === 'success') {
         let {data: {email, name, picture: {data: {url}}}} = await axios.get(`https://graph.facebook.com/me?fields=id,name,email,picture&access_token=${token}`);
```

Comment:
This line above is the same as before. Once we log in successfully, grab their info like email etc.